src/buffer.h

```diff
@@ -27,57 +27,56 @@
 #include "util.h"
 #include "global.h"
 
-
 /* Definitions ****************************************************************/
 // number of simulation network jitter buffers for evaluating the statistic
 // NOTE If you want to change this number, the code has to modified, too!
-#define NUM_STAT_SIMULATION_BUFFERS                 10
```

Comment:
There's an align setting that tells the formatter to align preprocess arguments across blank lines and comments, I think?  It might be worth trying.  Not particularly fussed on it, though.